app/src/main/java/com/trianguloy/urlchecker/modules/list/TextInputModule.java

```diff
@@ -59,8 +61,9 @@ public int getLayoutId() {
 
     @Override
     public void onInitialize(View views) {
-        edtxt_url = views.findViewById(R.id.url);
-        edtxt_url.addTextChangedListener(new DefaultTextWatcher() {
+        texttxt_url = views.findViewById(R.id.url);
+        edtxt_url = views.findViewById(R.id.urlEdit);
+        texttxt_url.addTextChangedListener(new DefaultTextWatcher() {
```

Comment:
This is the important bug, otherwise editing doesn't work!

```suggestion
        edtxt_url.addTextChangedListener(new DefaultTextWatcher() {
```

Reply:
Thank you, I will apply it as soon as I can!